code/ATMOSPHERICS/atmospherics.dm

```diff
@@ -273,4 +274,9 @@ Pipelines + Other Objects -> Pipe network
 
 
 /obj/machinery/atmospherics/proc/can_crawl_through()
-	return 1
\ No newline at end of file
+	return 1
+
+/obj/machinery/atmospherics/proc/returnPipenets()
+	. = list()
+	for(DEVICE_TYPE_LOOP)
+		. += returnPipenet(NODE_I)
```

Comment:
WTF dude? Make this proc empty at the parent level; put all this code under `obj/machinery/atmospherics/components/returnPipenets()`